spec/features/consumer/account/settings_spec.rb

```diff
@@ -27,5 +31,16 @@
       click_link I18n.t('spree.users.show.tabs.settings')
       expect(page).to have_content I18n.t('spree.users.show.unconfirmed_email', unconfirmed_email: 'new@email.com')
     end
+
+    it "allows the user to change their password" do
+      fill_in 'user_password', with: 'NewPassword'
+      fill_in 'user_password_confirmation', with: 'NewPassword'
+
+      click_button I18n.t(:update)
+      expect(find(".alert-box.success").text.strip).to eq "#{I18n.t(:account_updated)} ×"
```

Comment:
`find(".alert-box.success")` fails. It looks like the user is logged out after hitting Update.